tests/integ/ml/test_transformer.py

```diff
@@ -548,3 +551,144 @@ def test_binarizer(session):
     Utils.check_answer(
         res2, [Row(A=0, B=1, C=1), Row(A=1, B=0, C=1), Row(A=0, B=None, C=None)]
     )
+
+
+def test_normalizer(session):
+    df = session.create_dataframe(
+        [[1, 4, 5], [2, 4, 5], [5, 3, 6]], schema=["a", "b", "c"]
+    )
+    normalizer = Normalizer()
+    normalizer.input_cols = ["A", "B"]
+    normalizer.output_cols = ["res_A", "res_B"]
+    model = normalizer.fit(df)
+
+    # l2 normalization
+    res = model.transform(df)
+    Utils.check_answer(
+        res,
+        [
+            Row(A=1, B=4, C=5, RES_A=0.18257418583505536, RES_B=0.6246950475544243),
+            Row(A=2, B=4, C=5, RES_A=0.3651483716701107, RES_B=0.6246950475544243),
+            Row(A=5, B=3, C=6, RES_A=0.9128709291752769, RES_B=0.4685212856658182),
+        ],
+    )
+
+    normalizer = Normalizer(norm="max")
+    normalizer.input_cols = ["A", "B"]
+    normalizer.output_cols = ["res_A", "res_B"]
+    model = normalizer.fit(df)
+    # max normalization
+    res = model.transform(df)
+    Utils.check_answer(
+        res,
+        [
+            Row(A=1, B=4, C=5, RES_A=0.200000, RES_B=1.000000),
+            Row(A=2, B=4, C=5, RES_A=0.400000, RES_B=1.000000),
+            Row(A=5, B=3, C=6, RES_A=1.000000, RES_B=0.750000),
+        ],
+    )
+
+
+def test_label_encoder(session):
+    df_label = session.create_dataframe([1, 2, 3, 4, 5, 6])
+    df_inverse_label = session.create_dataframe(["aa", "bb", "cc"])
+    df = session.create_dataframe(
+        [[1, 4, 5], [2, 4, 5], [5, 3, 6]], schema=["a", "b", "c"]
+    )
+    df_unkown = session.create_dataframe(
+        [[1, 4, 5], [2, 4, 5], [5, 7, 6]], schema=["a", "b", "c"]
+    )
+    df_inverse = session.create_dataframe([[0], [2], [1]], schema=["a"])
+    labelencoder = LabelEncoder()
+    labelencoder.input_cols = ["A", "B"]
+    labelencoder.output_cols = ["res_A", "res_B"]
+    model = labelencoder.fit(df_label)
+
+    res = model.transform(df)
```

Comment:
can you use this model do inverse_transform() as well, and also unknown values for inverse_transform().